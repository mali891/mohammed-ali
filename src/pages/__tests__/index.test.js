import React from 'react'
import { render } from '@testing-library/react'

import Homepage from '../blog'

const props = {
  data: {
    allMarkdownRemark: {
      edges: []
    }
  }
}

describe('Homepage', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Homepage {...props} />)
    expect(asFragment()).toMatchSnapshot()
  })
})
