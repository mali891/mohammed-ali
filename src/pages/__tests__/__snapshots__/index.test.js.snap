// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage renders correctly 1`] = `
<DocumentFragment>
  <div
    class="u-anim--fade-in"
  >
    <header
      class="c-header"
      css="[object Object]"
    >
      <div
        class="c-header__container c-container"
        css="[object Object]"
      >
        <h1
          class="c-heading c-heading--xl c-heading__no-margin"
          css="[object Object]"
        >
          Blog
        </h1>
      </div>
      <button
        class="c-header__more-link c-link u-clr--secondary"
        css="[object Object]"
      >
        More
      </button>
    </header>
    <div
      class="c-header-scroll"
      css="[object Object]"
    />
    <main
      class="c-main"
    >
      <section
        class="c-content-group"
        css="[object Object]"
      />
    </main>
  </div>
</DocumentFragment>
`;
